OutOfSchool/OutOfSchool.ExternalFileStore.Api/OutOfSchool.ExternalFileStore.Api.csproj

```diff
@@ -21,5 +21,9 @@
         <RootNamespace>OutOfSchool.ExternalFileStore</RootNamespace>
     </PropertyGroup>
 
+    <ItemGroup>
```

Comment:
Api is for Api (interfaces and public objects), no need to put implementation dependency (Minio) here.